test/Plutip.purs

```diff
@@ -567,6 +569,176 @@ suite = do
           bsTx <- signTransaction =<< liftedE (balanceTx ubTx)
           submitAndLog bsTx
 
+    test "runPlutipContract: mustProduceAtLeast success" do
```

Comment:
I think most of the code of the test can be put in a new function and use it to short the size of all of the tests.

This would allow the test to just look like 

```
makeTest FirstConstraintValue SecondConstraintValue =
    do
    (firstTx /\ secondTx) <- newFunction FirstConstraintValue SecondConstraintValue
    testWith someFunction  firstTx secondTx
  where
    someFunction = ...
```

improving readability.

Reply:
@uhbif19 please don't do this. These tests are meant to be copy-paste-able by the users. No need to make them less readable just for a few code lines saved. We don't optimise for that metric, plus, it's easier to read when it's straightforward (less identifiers to keep in mind).

Reply:
@Luis-omega @klntsky Yes I was thinking about parametrizing tests initially, but remembered that you said something lke this comment before.